tests/cpu_bugs/xen_domu_mitigation_test.pm

```diff
@@ -788,7 +816,7 @@ sub run {
     zypper_call 'in -y xmlstarlet expect sshpass';
 
     get_expect_script();
-    exec_testcases();
+    check_and_run();
```

Comment:
please see the caller name get_expect_script, so adding check_and_run function in here is not suitable, i think

Reply:
The caller name is run() and I the check_and_run function is the refactor to exec_testcases which can avoid taking too long because of some errors.

Reply:
You are right